docs/content/integrations/dbt/reference.mdx

```diff
@@ -570,6 +570,22 @@ snapshots:
     - "{{ dagster.log_column_level_metadata() }}"
 ```
 
+Column dependencies can be removed from materialization metadata by disabling the collection of parent relation metadata. This can be done by setting the `enable_parent_relation_metadata_collection` argument to `False` in the `dagster.log_column_level_metadata()` macro:
```

Comment:
Feel free to ignore if you feel otherwise, but `collect_parent_relation_metadata` is shorter and might be equally clear.